kube/src/main/java/org/openshift/ping/kube/KubePing.java

```diff
@@ -195,8 +195,8 @@ public void destroy() {
         try {
             pods = client.getPods(_namespace, _labels);
         } catch (Exception e) {
-            if (log.isWarnEnabled()) {
-                log.warn(String.format("Problem getting Pod json from Kubernetes %s for cluster [%s], namespace [%s], labels [%s]; encountered [%s: %s]",
+            if (log.isTraceEnabled()) {
```

Comment:
I was wondering about failures after a successful connection, but then I realized we are talking about a transport protocol that communicates with the Kubernetes apiserver (the OpenShift master node itself). If something goes wrong, not only EAP clustering will stop working but the whole Openshift cluster administration. That would be considered a catastrofic issue and imho it's not openshift-ping responsibility.